src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -152,7 +155,7 @@
             <textarea
               id="textarea-message"
               v-auto-resize
-              class="font-16 font-italic ml-8"
+              class="font-14 ml-6 pr-5 mb-n2"
```

Comment:
The textarea adds some whitespace below the text itself, which you can see here:

![image](https://github.com/user-attachments/assets/d0298c3a-df20-4722-b8f8-5c5d54367e14)

`mb-n2` pulls up the bottom of the textarea ("shrinks it") to remove the whitespace:

![image](https://github.com/user-attachments/assets/daddfff0-c05a-49f7-98c2-d6efa2c1b600)